src/asgen/extractor.d

```diff
@@ -200,6 +201,24 @@ public:
                 gres.updateComponentGCID (cpt, ddata);
         }
 
+        if (pkg.gst.isNotEmpty) {
+            auto data = appender!string;
+            auto cpt = new Component ();
+
+            data.reserve(512);
+
+            cpt.setId (pkg.name);
+            cpt.setKind (ComponentKind.CODEC);
+            cpt.setName ("GStreamer Multimedia Codecs", "C");
```

Comment:
Not translated - not specific to the package in use. I don't know how to get anything better for here.